streaming/emr-eks-flink/data.tf

```diff
@@ -0,0 +1,64 @@
+
+data "aws_eks_cluster_auth" "this" {
+  name = module.eks.cluster_name
+}
+
+
+data "aws_ecrpublic_authorization_token" "token" {
+  provider = aws.ecr
+}
+
+data "aws_availability_zones" "available" {}
+
+
+
+data "aws_caller_identity" "current" {}
+
+
+
+# This data source can be used to get the latest AMI for Managed Node Groups
+data "aws_ami" "x86" {
+  owners      = ["amazon"]
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["amazon-eks-node-${module.eks.cluster_version}-*"] # Update this for ARM ["amazon-eks-arm64-node-${module.eks.cluster_version}-*"]
+  }
+}
+
+
+#---------------------------------------------------------------
+# Example IAM policy for Flink job execution
+#---------------------------------------------------------------
+data "aws_iam_policy_document" "flink_operator" {
```

Comment:
If this policy is created for running Flink Jobs, then don't call it as `flink_operator` for this resource. Just call it as `flink_sample_job`

Reply:
move this policy to `flink-team.tf`